src/sweeper/address_parser.py

```diff
@@ -147,6 +160,11 @@ def normalize_street_type(type_text):
 
     raise InvalidStreetTypeError(type_text)
 
+def normalize_street_name_pre_type(text):
+    '''normalizes highways by doing things like replaces SR with HWY and removes US
+    '''
+    return text.replace('SR', 'HWY').replace('US ', '')
```

Comment:
will these always be upper case? There are instances where the highway is formatted like `U.S.`

Reply:
In our data they will always be upper case but if it is run against a county's data there is a good chance of tile case.

Reply:
we'll need to make these replacements case insensitive then right?

Reply:
At this point in the code, `usaddress` has already normalized out `.`s and casing so we don't need to worry about it.